experiment_helpers/check_rmsThresh.m

```diff
@@ -55,14 +58,16 @@
             rmsValue = mean(data.rms(onset:offset, 1));
 
         % if no ost tracking, use RMS data to find onset/offset
-        elseif ~any(data.ost_stat >= 1) && any(data.rms(:, 1) > 0.03)
-            onset = find(data.rms > 0.01, 1, 'first') + 5;
+        elseif any(data.rms(:, 1) > 0.03)
+            % These values are more lax versions of the settings
+            % in free-speech\experiment_helpers\measureFormants.ost. They
+            % are relatively imprecise and would benefit from more nuance in the future.
+            onset = find(data.rms > 0.03, 1, 'first') + 5;
             offset = find(data.rms(:, 1)<0.03 & data.rms(:, 1)>0.02 & data.rms_slope<0, 1, 'first') - 5;
-
-            % use middle 80%
-            onset = floor(onset + ((offset-onset)/10));
-            offset = ceil(offset - ((offset-onset)/10));
-            rmsValue = mean(data.rms(onset:offset, 1));
+            if isempty(offset)
+                offset = min(onset+10,length(data.rms));
+            end
+            rmsValue = mean(data.rms(onset:offset));
```

Comment:
If data.rms has multiple columns, wouldn't we still want the "1" here?
```suggestion
            rmsValue = mean(data.rms(onset:offset, 1));
```

Reply:
Oops yes, thanks.